src/Microsoft.Azure.SignalR.AspNet/Constants.cs

```diff
@@ -7,8 +7,14 @@ internal static class Constants
     {
         public static class ClaimType
```

Comment:
Shall we move this class to the `Common` project? AspNetCore SignalR can use the same claim types.

Reply:
Sure, please check out https://github.com/Azure/azure-signalr/pull/164